source/app.d

```diff
@@ -38,6 +42,8 @@ void startServer(HTTPServerSettings settings)
     settings.bindAddresses = ["0.0.0.0"];
     settings.options = HTTPServerOption.defaults & ~HTTPServerOption.parseJsonBody;
 
+    lastFullPRCheck = SysTime(Date(0, 1, 1));
```

Comment:
Supposed to work, and Duration can represent any differences.
https://dlang.org/library/std/datetime/sys_time.min.html
```d
import std.datetime;
Clock.currTime - SysTime.min;
```
-1419840 weeks, -3 days, -2 hours, -21 minutes, -37 secs, -762 ms, -353 μs, and -8 hnsecs